rolocController.cpp

```diff
@@ -523,12 +538,41 @@ void ROLOCcontroller::onFreqSetTimerExpired()
     mpFreqencySetTimer->stop();
 }
 
+/**
+ * @brief ROLOCcontroller::rolocSetDepthMode - set depth mode
+ * @param mode - mode enum
+ */
+void ROLOCcontroller::rolocSetDepthMode(ROLOC::eLINEFINDER_MODE mode)
+{
+    Q_UNUSED(mode);
+    qint16 depthPacket = 0;
+    depthPacket = 0x0200;
+    depthPacket |= mFrequency;
```

Comment:
We can simplify this to one line
```suggestion
    qint16 depthPacket = 0x0200 | mFrequency;
```

But what is the 0x0200 ? Can we add a constant for that ? its a magic number at this point